src/dlgPackageExporter.cpp

```diff
@@ -274,8 +274,8 @@ void dlgPackageExporter::slot_export_package()
     QDir dir(tempDir);
     QStringList contents = dir.entryList();
     for (int i = 0; i < contents.size(); i++) {
-        QString fname = contents[i];
-        if (fname == "." || fname == "..") {
+        QString moduleFile = contents[i];
+        if (moduleFile == "." || moduleFile == "..") {
```

Comment:
This `if` can be removed if line 275 is changed to:
```
    QStringList contents = dir.entryList(QDir::NoDotAndDotDot);
```
However since the code here will die horribly if *__any__ directories* are fed to `zip_file_add` it may perhaps be wise to actually use:
```
    QStringList contents = dir.entryList(QDir::Files);
```
at least until the code is fixed to handle modules with sub-directories...
```